src/components/AddBook.js

```diff
@@ -11,28 +11,33 @@ function AddBooks() {
     const newBook = {
       item_id: uuidv4(),
       title: e.target.title.value,
-      category: e.target.author.value,
+      category: e.target.category.value,
     };
 
     dispatch(postBooks(newBook, 'POST'));
     e.target.title.value = '';
-    e.target.author.value = '';
+    e.target.category.value = '';
   };
 
   return (
-    <div className="Books">
+    <div className="submit-book">
       <form onSubmit={submitBookToStore}>
-        <input
-          type="text"
-          placeholder="Add Book"
-          name="title"
-        />
-        <input
-          type="text"
-          placeholder="Add Author"
-          name="author"
-        />
-        <button type="submit">Add Book</button>
+        <h3>ADD NEW BOOK</h3>
```

Comment:
- [ ] Please, add boldness to the ADD NEW BOOK title, this will improve the way to look the style. 🌴

Reply:
Done!